src/api.js

```diff
@@ -70,13 +82,17 @@ export class WalletApi {
         .catch(this._handleError)
   }
 
+  // This is overriding the native `client.subscribe` because it lacks support for `params`
   async subscribeToNotifications(params, cb) {
-    return this._callApiMethod('subscribe_notifications')([params.sessionId])
+    return this._callApiMethod('rpc.on')([params.session_id])
       .then(_ => {
         this.client.on('notifications', cb)
```

Comment:
As I ask below, what do you think about this?
```suggestion
        this.client.on('notifications', ([notification]) => cb(notification))
```